src/libs/record/record.c

```diff
@@ -0,0 +1,111 @@
+#include <kdbprivate.h>
+#include <kdbrecord.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+const char * const recordNamespaces[] = {
+	"dir:",
+	"user:",
+	"system:",
+};
+
+const size_t numRecordNamespaces = sizeof (recordNamespaces) / sizeof (char *);
+
+static void elektraRecordMount (void)
+{
+	// HACK! Replace with mounting API when it is finished
+	char buf[2048] = "";
+	for (size_t i = 0; i < numRecordNamespaces; i++)
+	{
+		snprintf (buf, 2047, "kdb mount record-session.cfg %s%s dump", recordNamespaces[i], ELEKTRA_RECORD_SESSION_KEY);
```

Comment:
@markus2330 should we be able to mount and unmount the recording session storage or should we just hard-code it in `kdbOpen`, as we already do for `system:/elektra`?

Personally, the more I think about it, I lean towards hardcoding those mount points. I can't think of a single reason why being able to mount and unmount the storage is beneficial.

Reply:
Yes, I agree. It should be hard-coded like the rest in /elektra. What about /elektra/record/session (e.g. system:/elektra/record/session)?